apps/src/templates/instructions/DynamicInstructions.jsx

```diff
@@ -92,6 +92,7 @@ class DynamicInstructions extends React.Component {
           {Object.keys(this.props.dynamicInstructions).map(key => {
             return (
               <div
+                className={this.props.dynamicInstructionsKey}
```

Comment:
I think we should prefix this with something, e.g. "dynamic-instruction-", to avoid the rare but possible collision with another CSS identifier on the page.  Also, because these are unique keys, we should be able to use ID instead of class.

Reply:
Added in the prefix!

Since all of the dynamic instructions are still in the DOM, wouldn't we want to keep them as classes? 
<img width="666" alt="Screen Shot 2021-07-14 at 5 29 15 PM" src="https://user-images.githubusercontent.com/40412372/125695729-8188796d-56d9-4a74-b675-702b30daf0ba.png">

Reply:
In this example, I think they are mislabeled?  `"-selectDataset"` should only be for the dynamic instruction that has the `"selectDataset"` key, not all of them.  Once they are each unique, then ID seems more appropriate.